clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts

```diff
@@ -26,7 +26,7 @@ export interface DependenciesDesc {
 export function loadDepFile(depFileUri: vscode.Uri): DependenciesDesc {
     const copybooks: string[] = fs.readFileSync(depFileUri.fsPath).toString().split("\n")
         .filter(e => e.trim().length > 0)
-        .map(e => e.trim());
+        .map(e => e.trim().substring(0, 8));
```

Comment:
nice, we should also throw an error from LSP side once the user tries to put copybook name bigger than 8 chars, what do you think?

Reply:
@abdrabrcom is working on it

Reply:
The LSP should show a warning, the extension should truncate silently AFAIK.

Reply:
yesterday we agreed about **severity:warning**